projects/batfish/src/main/java/org/batfish/job/BatfishJobExecutor.java

```diff
@@ -41,105 +50,143 @@ public BatfishJobExecutor(
     _description = description;
   }
 
+  /**
```

Comment:
(comments at the top of diff apply to code above them -- GitHub won't let me comment up there.)

Add class-level javadoc with a simple, non-jargony explanation of what this class does.

Delete unused constants (e.g., JOB_POLLING_PERIOD)?

Reply:
done